$MirOS: ports/www/lynx/patches/patch-WWW_Library_Implementation_HTAccess_c,v 1.1 2010/12/23 20:25:30 tg Exp $

	Fix spelling; suggested for upstream inclusion.

--- WWW/Library/Implementation/HTAccess.c.orig	Thu Feb 23 02:03:09 2012
+++ WWW/Library/Implementation/HTAccess.c	Sat Mar  3 18:20:20 2012
@@ -26,7 +26,7 @@
  *		  on the local host (because HTLoadFile() doesn't try ftp
  *		  for those) and will substitute ftp for remote files. FM
  *	28 Jan 95 Tweaked PDM's proxy override mods to handle port info
- *		  for news and wais URL's. FM
+ *		  for news and wais URLs. FM
  *
  *  Bugs
  *	This module assumes that that the graphic object is hypertext, as it
@@ -995,7 +995,7 @@ static BOOL HTLoadDocument(const char *f
 	 * subsequent access attempts.  We'll check recursively, and retrieve
 	 * the final URL if we had multiple redirections to it.  If we just
 	 * went to HTLoad now, as Lou originally had this, we couldn't do
-	 * Lynx's security checks and alternate handling of some URL types. 
+	 * Lynx' security checks and alternate handling of some URL types. 
 	 * So, instead, we'll go all the way back to the top of getfile in
 	 * LYGetFile.c when the status is HT_REDIRECTING.  This may seem
 	 * bizarre, but it works like a charm!  - FM
